cardano-constitution/data/defaultConstitution.schema.json

```diff
@@ -22,34 +30,33 @@
+                     , "$comment": "This restricts the parameter 9000 to be at least 0"
                  },
                  "9001": {
                      "predicates": { "minValue": 0, "maxValue": 0 }
-                     , "____comment____": "The 9001 should be least 0 and at most 0 ==> can only be 0"
-                     , "____comment____": "A hard/frozen restriction on the parameter, because it cannot change as long as constitution remains the same"
+                     , "$comment": "The 9001 should be least 0 and at most 0 ==> can only be 0"
+                     , "$comment": "A hard/frozen restriction on the parameter, because it cannot change as long as constitution remains the same"
                  },
                  "9002": {
                      "predicates": { "minValue": 1, "maxValue": 0 }
-                     , "____comment____": "The 9002 should be least 1 and at most 0. UNSATISFIABLE"
-                     , "____comment____": "A non-sense parameter restriction. A proposal trying to touch this parameter will always fail"
-                     , "____comment____": "Currently, we don't have a guard against configuring a constitution with such non-sense restrictions"
-                 },
-                 "9003": {
-                     "predicates": {  }
-                     , "____comment____": "The 9003 does no contain any restrictions, and thus is allowed to any take integer Value"
-                     , "____comment____": "Any parameter that is not specified in a constitution-config, will make its change-proposal fail."
-                     , "____comment____": "This empty-predicates config, acknowledges the existence of a parameter and allows it to be changed (to any value)."
+                     , "$comment": "The 9002 should be least 1 and at most 0. UNSATISFIABLE"
+                     , "$comment": "A non-sense parameter restriction. A proposal trying to touch this parameter will always fail"
+                     , "$comment": "Currently, we don't have a guard against configuring a constitution with such non-sense restrictions"
                  },
                  "9003": {
                      "predicates": { minValue: 1 , minValue: -1 }
-                     , "____comment____": "VERY IMPORTANT: json schema (and JSON) allows duplicate object entries:"
-                     , "____comment____": "e.g. duplicate parameter entries or duplicate predicates."
-                     , "____comment____": "Only one of the duplicates will be *silently* loaded and we don't guarantee the order"
-                     , "____comment____": "SO TAKE CARE OF DUPLICATES"
+                     , "$comment": "VERY IMPORTANT: json schema (and JSON) allows duplicate object entries:"
+                     , "$comment": "e.g. duplicate parameter entries or duplicate predicates."
+                     , "$comment": "Only one of the duplicates will be *silently* loaded and we don't guarantee the order"
```

Comment:
There are multiple guardrails that relate to the same parameter and the same bounds (like cannot be below 30 and has to be positive). Are we sure we can just discard the redundant ones?